gammapy/stats/tests/test_poisson.py

```diff
@@ -37,6 +38,9 @@ def test_excess_error():
     assert_allclose(excess_error(n_on=10, n_off=20, alpha=0.1), 3.1937439)
     assert_allclose(excess_error(n_on=4, n_off=9, alpha=0.5), 2.5)
 
+def test_Helene_ULs():
+    assert_allclose(Helene_ULs(excess=50, error=40, conf_level=99.73), 119.708038)
```

Comment:
I get a different value in the test?!
@bkhelifi - does the test pass for you locally?
```
python setup.py test -V -t gammapy/stats/tests/test_poisson.py
```
```
_____________________________________________________________________________ test_Helene_ULs _____________________________________________________________________________

    def test_Helene_ULs():
>       assert_allclose(Helene_ULs(excess=50, error=40, conf_level=99.73), 119.708038)
E       AssertionError: 
E       Not equal to tolerance rtol=1e-07, atol=0
E       
E       (mismatch 100.0%)
E        x: array(162.727686)
E        y: array(119.708038)

gammapy/stats/tests/test_poisson.py:42: AssertionError
============================================================= 1 failed, 18 passed, 1 xfailed in 1.14 seconds ==============================================================
```

Reply:
It looks like the paper has plenty of "reference values" on what the result should be?
Could you pick one of those and leave a one line comment in the test saying where to find the test case in the paper (e.g. Table 1, 2 or 3?)

Reply:
I did a stupid typo. Thanks a lot to have re-check that. Corrected.
I have added a test to make a wider coverage of the code